lnwallet/channel.go

```diff
@@ -1467,18 +1459,6 @@ func (lc *LightningChannel) createStateHintObfuscator() {
 	}
 }
 
-// Stop gracefully shuts down any active goroutines spawned by the
-// LightningChannel during regular duties.
-func (lc *LightningChannel) Stop() {
```

Comment:
Nit: looks like we can also remove the now unused `shutdown` and `quit` members.